docs/source/installation/for_dev.rst

```diff
@@ -0,0 +1,126 @@
+Manim Installation For Developers
+=================================
+
+	This documentation is for developers who wats to contribute to ``ManimCommunity/manim``.
+
```

Comment:
```suggestion
Manim community uses Poetry (instead of pip). We chose poetry because blah blah blah
```

Reply:
I am not gonna write this.

Reply:
As you wish. But I think it would be of great help since poetry is not the standard. Explaining why we use a non-standard tool for development makes sense to me, and I would appreciate the explanation if I were the reader.

Reply:
No, wait you can't it as non-standard tool because it internally uses pip and it is a dependency management tools. See https://packaging.python.org/tutorials/managing-dependencies/ . Maybe if you ask me I will link to this.

Reply:
When I say "non-standard" I mean that it is not the usual way of doing things. It is interesting that the official python documentation links and recommends poetry. But, this line 

```
Developers of Python libraries, or of applications that support distribution as Python libraries, should also consider the poetry project as an alternative dependency management solution.
```

says that it is _an alternative_. I assume that the vast majority of devs trying to contribute to manim will not have heard of poetry. It would be helpful if we left a link (if perhaps not a full explanation) for them :)